src/main/xml/options/soot_options.xml

```diff
@@ -405,6 +405,16 @@
                 and the original classes you loaded. Use this option at your own risk.
             </long_desc>
         </boolopt>
+        <boolopt>
+            <name>Weak Map Structures</name>
+            <alias>weak-map-structures</alias>
+            <short_desc>Replace some Arrays with Weak Map structures</short_desc>
```

Comment:
Could we make the short_desc a bit more expressive? In the options html, only the short_desc is shown for general options, so this is the most important part.

Maybe something like: "Use weak references in Scene to prevent memory leakage when removing many classes/methods/locals".

Reply:
Fixed